### PR TITLE
feat: apply test code for collector list at collector main page

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA
image

### CODE REVIEW

**Review by piggggggggy — APPROVED** [review]
LGTM!!!

**Review by yuda110 — APPROVED** [review]
LGTM!!!!

File: apps/web/tests/asset-inventory/collector/collector-main/collector-list.spec.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    /* Init */
+    test.beforeEach(async ({ browser }) => {
+        page = await browser.newPage();
+        await goToCollectorPage();
+    });
+
+    test.afterEach(async () => {
+        await page.close();
+    });
+
```
Comment (sulmoJ) [inline apps/web/tests/asset-inventory/collector/collector-main/collector-list.spec.ts:19]: Praise: Thank you for nice example.

**Review by sulmoJ — APPROVED** [review]
LGTM!

**Pull request merged**